packages/@okta/vuepress-site/docs/reference/core-okta-api/index.md

```diff
@@ -277,12 +277,14 @@ Most of the operators listed in the [SCIM Protocol Specification](https://www.rf
 | `ne`       | not equal             | If the attribute value does not match the operand value, there is a match. |
 | `pr`       | present (has value)   | If the attribute has a non-empty value or if it contains a non-empty node for complex attributes, there is a match.                                                                                                                                                           |
 | `sw`       | starts with           | The entire operand value must be a substring of the attribute value that starts at the beginning of the attribute value. This criterion is satisfied if the two strings are identical.                                                                                         |
+| `co`       | contains              | If the operand value exists as a substring in the attribute value, there's a match. This includes the case where the attribute and operand values are identical.                       |
```

Comment:
I'm hesitant to add this operator in the table ... because it's only used by Systems Log, and now Users and Groups. I looked into adding this operator to this table about a month and a half ago and it was decided just to add the note below. I'd need to look into the ticket, but I believe this was an engineering decision, not mine.